source/features/hide-zero-packages.tsx

```diff
@@ -0,0 +1,21 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+function init(): void {
+	const packageElem: HTMLElement | null = select(`.numbers-summary a[href='/${getRepoURL()}/packages']`);
+	if (packageElem && packageElem.textContent!.trim().startsWith('0')) {
+		packageElem.parentElement!.remove();
+	}
+}
+
+features.add({
+	id: __featureName__,
+	description: 'Hides the `Packages` tab in repositories if it’s empty.',
+	screenshot: 'https://user-images.githubusercontent.com/35382021/62420784-a6652f00-b687-11e9-82a5-eaf6a636f218.png',
```

Comment:
What @notlmn meant here is that you can copy-paste the exact “before after” text so all screenshots will have the same style. Also you can take the screenshot after zooming in the page at 200%, if your screen can fit that, so the image will be in higher quality

Reply:
Sorry, I don't know how to deal with the image. I'm using a very small laptop.

Reply:
That’s ok, it looks good as is 🙂 👍